portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,159 @@
 #content {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
 }
 
 #greeting-container {
+  margin-top: 3px;
+  color:white;
+  size: 20px;
+  background: gray; 
+}
+
+.image-thumbnails {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.image-thumbnails img {
+  margin: 10px;
+  max-height: 200px;
+  max-width: 200px;
+}
+
+.image-thumbnails img:hover {`
+  border: thick solid blue;
+}
+
+#random-image-container img {
   margin-top: 20px;
 }
+
+body {
+  font-family: Arial;
+  background-image:linear-gradient(rgba(0,0,0,0.2),rgba(0,0,0,0.2)),url(images/back.jpg);
+}
+
+.wrapper {
+  display: grid;
+    grid-template-columns: 1000px 1000px 1080px;
+  grid-auto-rows: 200px;
+}
+*
+{
+    margin: 0;
+    padding: 0;
+}
+
+.main-nav
+{
+	margin-top:100px;
+	float:right;
+	font-size: 15px;
+}
+
+.main-nav li
+{
+    display: inline-block;
+    padding: 10px 10px;
+    
+}
+
+.main-nav li.active 
+{
+	background: gray;  
+}
+
+.main-nav li:hover
+{
+    	background: gray;
+		padding: 10px 10px;
+}
+
+h1
+{
+    color: white;
+    font-size: 100px;
+    text-align: center;
+    margin-top: 300px;
+}
+
+header
+{
+    background-image:linear-gradient(rgba(0,0,0,0.2),rgba(0,0,0,0.2)),url(images/back.jpg);
+	height: 100vh;
+}
+
+footer
+{
+    background-color: gray;
+}
+
+.button
+{
+    margin-top: 30px;
+    margin-left: 900px;
+}
+
+.row
+{
+    max-width: 2000px;
+    margin: auto;
+}
+.gallery{ 
+    margin:10px 50px;
+}
+.gallery img{
+    transition:0.5s;
+    padding: 15px;
+    max-width: 300px;
+    max-height: 300px;
+    filter: grayscale(100%);
+}
+.gallery img:hover{
+    filter:none;
+    transform: scale(1.1);
+}
+p{
+    color:white;
+    font-size:30px;
+    font-weight: 700;
+    background-color: gray;
+}
+dt{
+    color:white;
+    font-size:30px;
+    font-weight: 700;
+    background-color: gray;
+}
+dl{
+    color:white;
+    font-size:30px;
+    font-weight: 700;
+    background-color: gray;
+}
+h2{
+    color:black;
+    background-color: gray;
+    font-size: 50px;
+    text-align:left;
+}
+
+button {
+  background-color:green;
+  color: white;
+  padding: 15px 32px;
+  text-align: center;;
+  display: inline-block;
+  font-size: 16px;
+  margin: 4px 2px;
+  cursor: pointer;
+}
+
```

Comment:
NIT: you can remove the extra blank lines.  Typically files just have 1 new line at the end (like your index.html)

Reply:
fixed